lib/index.js

```diff
@@ -38,6 +38,8 @@ exports.balm = (opts = {}) => {
 
     traversy(root, fileExtRegexStr, bundleFile);
 
+    addHmrToBundles(port, blocksDir);
```

Comment:
I'm still confused why we put this in every time, when we don't want hmr in there in non dev mode.